public/index.html

```diff
@@ -24,7 +24,18 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <link href="https://fonts.googleapis.com/css?family=Lato:400,700,900&display=swap" rel="stylesheet">
+    <link
+      rel="preload"
+      href="https://fonts.googleapis.com/css?family=Lato:400,700,900&display=swap"
+      as="style"
+      onload="this.onload=null;this.rel='stylesheet'"
```

Comment:
is this necessary? 🤔

Reply:
Yep: https://web.dev/defer-non-critical-css/#optimize

Reply:
Is the font non-critical? Doesn't this produce pop in?

Reply:
It produces a kind of pop at the very first time you open the page. Anyway, this is [recommended by google](https://web.dev/optimize-lcp/#reduce-css-blocking-time) because it improves the [Largest Contentful Paint](https://web.dev/lcp/#what-is-lcp) and thus the UX.